.github/workflows/e2e.yml

```diff
@@ -17,4 +17,5 @@ jobs:
           PUBLIC_ENV=test"
           echo "$variables" > .env
           yarn install
+          npx playwright install
```

Comment:
Why? :D

Reply:
This is to make sure we have new browsers, the CI failed with this before: 
<img width="983" alt="Screenshot 2023-08-20 at 14 46 59" src="https://github.com/streamchaser/streamchaser/assets/9653806/a1947b5e-f500-420c-a7b6-78903c86a470">

Reply:
Ahh okay